packages/react-jss/src/createUseStyles.js

```diff
@@ -27,79 +26,60 @@ const createUseStyles = <Theme: {}>(styles: Styles<Theme>, options?: HookOptions
       : // $FlowFixMe
         (): Theme => noTheme
 
+  const useLayoutEffect = typeof window === 'undefined' ? React.useEffect : React.useLayoutEffect
```

Comment:
we depend on https://github.com/cssinjs/jss/blob/master/packages/jss/package.json#L44 anyways